scripts/gitlab/publish-docs.sh

```diff
@@ -15,12 +16,20 @@ build_docs() {
     npm run build:markdown
 }
 
+build_config() {
+    echo "_______Build config docs______"
+    npm install
+    npm run generate-data
+    npm run generate-docs
```

Comment:
Seems that this script will actually always use latest commit from `master`. In future it might be better to just use the checked out `cli/mod.rs` instead of having `parity-config-generator` fetching another one from master.